### PR TITLE
Add support for PHP 8.0

### PR DESCRIPTION
**Pull request by jmsche**

### CODE REVIEW

File: src/IndexBuilder.php [hunk 1]
```diff
@@ -130,7 +130,7 @@ public function purgeOldIndices(string $indexName): array
 
         // Newest first
         uasort($indexes, function ($a, $b) {
-            return $a['date'] < $b['date'];
+            return $a['date'] < $b['date'] ? 1 : -1;
```
Comment (jdreesen) [inline src/IndexBuilder.php:133]: Why not using the spaceship operator?
```suggestion
            return $b['date'] <=> $a['date'];
```

Comment (jmsche) [reply src/IndexBuilder.php]: Thanks, I always forget about this one :)

File: composer.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2|^8.0",
```
Comment (Korbeil) [inline composer.json:18]: Single pipe is not recommended and deprecated, you should use double pipe here.
```suggestion
        "php": "^7.2 || ^8.0",
```

It was in documentation before, but can't find where, so here is an issue about this https://github.com/composer/composer/issues/6755

Comment (jmsche) [reply composer.json]: TIL, thank you @Korbeil :)

**Review by lyrixx — APPROVED** [review]

**Review by Korbeil — APPROVED** [review]

**Comment by damienalexandre** [comment]
Thanks a lot everyone!

**Pull request merged**